compose/envvars-precedence.md

```diff
@@ -43,4 +41,4 @@ Description for each column:
 * Command line - Environmental variable passed via `docker compose run -e <KEY[=[VAL]]>`.
 * Resolved as - This column expresses the result available in the container.
 
-**Note that the "`OS`" has precedence over "`.env` file" column for variable resolution in the other columns.**
```

Comment:
This is the same sentence, right?

Reply:
It's the same. The only thing that changed is that it has a new line in the end of the file.
This is a standard (on POSIX systems at least) to have a file ending with a new line.